### DISCUSSION TITLE
Welcome to FieldsLinker Discussions!

### DISCUSSION THREAD

**Discussion by PhilippeMarcMeyer** [PhilippeMarcMeyer/FieldsLinker#28] [General]
## 👋 Welcome dear FieldsLinker users and fellow developpers!

Feel free to ask and suggest in this place

I will be glad to hear from you

All the best

Philip

**Comment by obrienb111** [comment]
Hi Philip

I was wondering if you had an example of how to upload a file (simple csv headers list) such that it can be loaded into a list?

FieldsLinker is great!

Regards,
BOB

**Reply by PhilippeMarcMeyer** [reply]
Hi Bob !

check https://github.com/PhilippeMarcMeyer/vanillaSelectBox

at the end of the readMe file you'll see an example function : doSearch() that loads a json file
that's not what you asked but that doas half of the job

of course it would be better to use a json file or the retrun json from a back-end that would read and parse you csv file

You could also keep the results of users choices to give tham back to them if they want to upload a file they already process with FieldsLinker

Let me know if you need more advice

Sincerely 

Philip

**Reply by PhilippeMarcMeyer** [reply]
if you need to let the user upload a csv file. thats another thing

**Comment by PhilippeMarcMeyer** [comment]
if you need to let the user upload a csv file check : https://github.com/PhilippeMarcMeyer/Words-Crisscrossing

**Comment by obrienb111** [comment]
Hi Philip

Many thanks for taking the time to reply. I will take a look and see what I can do. I’m not very good at this stuff!

Regards
Bob

**Comment by obrienb111** [comment]
Hi Philippe,

I am trying to build something that I think you would easily be able to do. I am a database developer and my web front end dev experience is very basic. 

Would you be interested in doing some freelance work? If so, I’ll explain further. 

Regards
Brendan





Sent from my iPhone

**Comment by PhilippeMarcMeyer** [comment]
Why not ?
If it's simple that could even be free

BTW I'm a former database developer :-)

Le jeu. 22 juil. 2021 à 16:18, obrienb111 ***@***.***> a
écrit :

**Comment by obrienb111** [comment]
Hi,

Fantastic. Let me put something together to send to you. I have started something purely as a proof-of-concept. I’ll forward details for that too.

Regards
Brendan


Sent from my iPhone

On 22 Jul 2021, at 19:17, philippe MEYER ***@***.***> wrote:

﻿

Why not ?
If it's simple that could even be free

BTW I'm a former database developer :-)

Le jeu. 22 juil. 2021 à 16:18, obrienb111 ***@***.***> a
écrit :

**Comment by int211** [comment]
Hi Philipp, say me please, is there a way to put more than 1 IN list to connect to the second, example: I have 2 lists with 5 fields at left and want to connect with one to the right, its possible with current release?
Thanks a lot and congrats for you job, at time, sorry my poor english.

**Reply by PhilippeMarcMeyer** [reply]
Hi Celso 
If you want to connect a field on the left side to several fields on the right side yes it is possible : check the manyToMany option
If it's something else tell me in your language I should be able to understand

**Reply by int211** [reply]
Obrigado Phillippe,
Então o que eu gostaria era de ter mais de duas listas que pudessem se conectar exemplo, do lado A eu ter 2 listas e do lado B eu ter uma lista grande, assim eu poderia alimentar a lista A e B para se conectar com a C o cenario seria:
sendo A e B com 3 itens e C com 6 itens e tratamento distinto entre elas.
A -> C 
B -> C.
Obrigado Phillipe.

**Reply by PhilippeMarcMeyer** [reply]
Sorry Celso,
That's too specific
I won't do it
You can still put your lists in a single left one with a prefix to distinguish them.
You can also use several instances of field linker

Cheers
